NEWS.md

```diff
@@ -1,3 +1,9 @@
+# messydates 0.4.1
+
+## Functions
+
+* Closed #83 by fixing how logical comparisons works for negative and year only dates
```

Comment:
Can we comment also here about how from 0.4.1 the required version of R has increased?